src/Features/Core/Portable/ExternalAccess/UnitTesting/API/UnitTestingDocumentSpan.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.EditAndContinue.Contracts;
+using Microsoft.CodeAnalysis.Navigation;
+
+namespace Microsoft.CodeAnalysis.ExternalAccess.UnitTesting.Api
+{
+    internal readonly struct UnitTestingDocumentSpan
+    {
+        internal UnitTestingDocumentSpan(DocumentSpan sourceSpan, FileLinePositionSpan span)
+        {
+            DocumentSpan = sourceSpan;
+            Span = span;
+        }
+
+        public DocumentSpan DocumentSpan { get; }
+        public FileLinePositionSpan Span { get; }
```

Comment:
could doc the difference between these two

Reply:
done!